### PR TITLE
first gartee commit

### PR DESCRIPTION
**Pull request by gartee-john-PFG**
making some really cool improvements

### CODE REVIEW

File: branchExercise.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+gartee line 1
+gartee line 2
```
Comment (SusmanNickiPFG) [inline branchExercise.txt]: Can you please add in a line 3 here? Thanks!

**Review by SusmanNickiPFG — CHANGES_REQUESTED** [review]
Need one change please, see comments

File: branchExercise.txt (hunk 1 above)
Comment (gartee-john-PFG) [reply branchExercise.txt]: will do

**Review by SusmanNickiPFG — APPROVED** [review]
Looks great, thanks for the update!

**Review by AndersonRyanPFG — APPROVED** [review]
You did great

**Review by RikerGregPFG — APPROVED** [review]
Looks good!

**Review by FarisDianePFG — APPROVED** [review]
This looks good.  Thanks.

**Review by BakerSaraPFG — APPROVED** [review]
Cool, nice job.

**Pull request merged**